tinacms/tinacms
# Handle root path

## Allow `path: ""` or `path: "/"`  in a collection. 


## How to test
In a collection in one of the examples projects make the path `/`, or `""`. You can update the match.include property so that the collection can still find the correct documents. 

Check admin collection list page and edit document page and make sure you can create update and delete documents. 

Fixes ENG-898
